adapters/adform/params_test.go

```diff
@@ -48,6 +48,10 @@ var validParams = []string{
 	`{"mid":"123","mkv":"color:"}`,
 	`{"mid":"123","mkw":"green,male"}`,
 	`{"mid":"123","mkv":" ","mkw":" "}`,
+	`{"mid":"123","cdims":"500x300:400x200","mkw":" "}`,
```

Comment:
The colon (:) separator doesn't work and instead a comma (,) should be used so: _"cdims":"500x300:400x200"_ should become _"cdims":"500x300,400x200"_